bindings/rust/s2n-tls-hyper/src/connector.rs

```diff
@@ -110,14 +115,22 @@ where
 pub struct Builder<Http, ConnBuilder> {
     http: Http,
     conn_builder: ConnBuilder,
+    insecure_http: bool,
 }
 
 impl<Http, ConnBuilder> Builder<Http, ConnBuilder> {
+    /// If enabled, allows communication with insecure HTTP endpoints in addition to secure HTTPS endpoints (default: false).
+    pub fn with_insecure_http(&mut self, enabled: bool) -> &mut Self {
```

Comment:
Thought: is `insecure` too strong a word for this? Maybe something like `with_plaintext_http` would be a bit more ambivalent.

Although I guess if you're going through the trouble of using the `s2n-tls-hyper` connector, plaintext http is a little sus. I don't think the current naming was bad, just curious about reasoning.

Reply:
Yeah `insecure` was meant less as a warning and more to differentiate from "secure" in HTTP**S**. So `with_plaintext_http` works just as well if that makes the meaning more clear.